src/lib/services/feature-toggle-service.ts

```diff
@@ -940,37 +940,29 @@ class FeatureToggleService {
         const result = await this.featureToggleClientStore.getClient(
             query || {},
         );
-        if (this.flagResolver.isEnabled('cleanClientApi')) {
-            return result.map(
-                ({
-                    name,
-                    type,
-                    enabled,
-                    project,
-                    stale,
-                    strategies,
-                    variants,
-                    description,
-                    createdAt,
-                    lastSeenAt,
-                    impressionData,
-                }) => ({
-                    name,
-                    type,
-                    enabled,
-                    project,
-                    stale,
-                    strategies,
-                    variants,
-                    description,
-                    createdAt,
```

Comment:
createdAt and lastSeenAt should not be exposed anymore according to openapi aschema